### DISCUSSION TITLE
phenotype_disease.tsv and phenotype_gene.tsv: built by script or not?

### DISCUSSION THREAD

**Discussion by jeremyjyang** [nih-cfde/published-documentation#333] [Q&A] [answered]
According to https://github.com/nih-cfde/published-documentation/wiki/submission-prep-script, the files phenotype_disease.tsv and phenotype_gene.tsv are built by the submission prep script. However, according to the script itself, https://osf.io/c67sp/, "These files should be present in submissionDraftDir before running this script."

```
targetTSVs = ( 'file.tsv', 'biosample.tsv', 'biosample_disease.tsv', 'biosample_gene.tsv', 'biosample_substance.tsv', 'subject_disease.tsv', 'subject_role_taxonomy.tsv', 'subject_substance.tsv', 'collection_disease.tsv', 'phenotype_disease.tsv', 'phenotype_gene.tsv' )
```

Please clarify.

**Comment by abradyIGS** [comment]
Thanks -- that is indeed an insane set of requirements as stated. will fix and push an update; watch this issue for announcement.

(the two tables you mention are to be built by the script; i deferred phenotype processing until end of month, but as long as you don't have any HPO terms i can make it create empty stubs as needed for compliance. if you do have HPO terms you want to use in the user-built tables [`subject_phenotype` and, in the 4/1 release, `collection_phenotype`], please let me know asap)

**Comment by abradyIGS** [comment, marked as answer]
This issue has been fixed and the script updated.

Ontology files and the submission prep script have all been updated and published [here](https://osf.io/bq6k9/): please overwrite local copies and try again.

Note: the master C2M2 JSON schema has also updated (to match previous development schema): it is [here](https://osf.io/c63aw/) and an up-to-date copy needs to be included with submissions during prep with the script.
